Source/Scene/ClippingPlaneCollection.js

```diff
@@ -149,6 +153,18 @@ define([
             }
         },
 
+        /**
+         * Gets the event fired when members of the collection have been added or removed
+         * @type {Event}
+         * @memberof ClippingPlaneCollection.prototype
+         * @readonly
+         */
+        collectionChanged : {
```

Comment:
This should probably follow the same pattern as other objects in Scene, such as ImageryLayerCollection/ModelAnimationCollection, which have specific `layerAdded`/`layerRemoved` events. We should also pass the plane and index as part of the event.

Reply:
👍 good suggestion thanks